### ISSUE TITLE
Updates page throws cannot read property 'version' of undefined

### ISSUE THREAD

**Issue by zolee** [metabase/metabase#9950]
I have upgraded from `0.32.5` to `0.32.7` and **Updates** page still broken for me (even tough the problem was set as solved in #9929 after the `0.32.6` broken build). If I click on the **Updates** menu item I can see the following error printout in the Chrome dev tools console:

```
app-main.bundle.js?a36a3806cede064f5de3:5 Uncaught TypeError: Cannot read property 'version' of undefined
    at t.value (app-main.bundle.js?a36a3806cede064f5de3:5)
    at t.value (app-main.bundle.js?a36a3806cede064f5de3:5)
    at u._renderValidatedComponentWithoutOwnerOrContext (vendor.bundle.js?a36a3806cede064f5de3:1)
    at u._renderValidatedComponent (vendor.bundle.js?a36a3806cede064f5de3:1)
    at u.performInitialMount (vendor.bundle.js?a36a3806cede064f5de3:1)
    at u.mountComponent (vendor.bundle.js?a36a3806cede064f5de3:1)
    at Object.mountComponent (vendor.bundle.js?a36a3806cede064f5de3:1)
    at Object.updateChildren (vendor.bundle.js?a36a3806cede064f5de3:1)
    at J._reconcilerUpdateChildren (vendor.bundle.js?a36a3806cede064f5de3:1)
    at J._updateChildren (vendor.bundle.js?a36a3806cede064f5de3:1)
```

Seems that all menu items work until you click on **Updates** that is why this has not identified sooner I think. After clicking on the broken **Updates** menu non of the setting page will load until you refresh the page.

A temporary workaround for using the settings is not to click on the Updates menu item.

### Environment
- Browser independent issue, could be reproduced on Chrome 74 / Firefox 66
- Windows 10
- Connected database is MSSQL Server2012, 
- Metabase version `0.32.7`
- Metabase hosting environment: Windows Server 2012
- Metabase internal database: H2 in memory database

### Reproduction steps
Go to the **Admin**, click on the **Updates** menu item on the **Settings** page.

**Comment by zolee** [comment]
#9929

**Comment by camsaul** [comment]
Fixed by #9959. We will be releasing this as part of 0.32.8 later today.
